src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1223,24 +1223,29 @@ because xbuild doesn't support framework reference assemblies.
 	</ItemGroup>
 </Target>
 
-<PropertyGroup>
-	<_ManagedUpdateAndroidResgenInputs>
-		$(MSBuildAllProjects);
-		@(AndroidResource);
-		@(AndroidBoundLayout);
-		@(_ReferencePath);
-		@(_LibraryResourceDirectoryStamps);
-		@(_AdditonalAndroidResourceCachePaths->'%(Identity)\cache.stamp');
-		$(_AndroidBuildPropertiesCache);
-		$(ProjectAssetsFile);
-	</_ManagedUpdateAndroidResgenInputs>
-</PropertyGroup>
+<Target Name="_BeforeManagedUpdateAndroidResgen">
+  <PropertyGroup>
+    <_ManagedUpdateAndroidResgenInputs>
+      $(MSBuildAllProjects);
+      @(AndroidResource);
+      @(AndroidBoundLayout);
+      @(_ReferencePath);
+      @(_LibraryResourceDirectoryStamps);
+      @(_AdditonalAndroidResourceCachePaths->'%(Identity)\cache.stamp');
```

Comment:
This might conflict with: https://github.com/xamarin/xamarin-android/pull/3908

You can probably go ahead and remove this line:
```suggestion
```